app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -200,6 +201,8 @@
     @include govuk-media-query($until: tablet) {
       grid-area: 1 / 1;
     }
+
+    margin-top: 2px;
```

Comment:
Why is this margin top needed? It seems to be trying to align the bottom of this text with the bottom of the link text next to it, but that's underlined so it's actually misaligning the text.

Reply:
have removed.... not convinced it aligned correctly